tests/Unit/Evolution/Systems/RadiationTransport/M1Grey/ComputeM1HydroCouplingJacobian.py

```diff
@@ -0,0 +1,391 @@
+# Distributed under the MIT License.
```

Comment:
I'm not convinced this python test is worth much.  It's obviously a copy-paste of the c++ source, so any math errors are going to be the same in both.  The IMEX `test_sector` will test this better.

Reply:
I think the fuzzy testing has a small value to check for edge cases producing 0s in denominators.  Nevertheless, I agree with the similarity between the .py and .cpp code, so the test_sector (mentioned below) will be included.